akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSupervisionSpec.scala

```diff
@@ -387,70 +387,6 @@ class InterpreterSupervisionSpec extends AkkaSpec with GraphInterpreterSpecKit {
       lastEvents() should be(Set(OnNext(25))) // 1 + 4 + 20
     }
 
-    "restart when Conflate `seed` throws" in new OneBoundedSetup[Int](Seq(Conflate(
```

Comment:
yes, you can move it there. You don't really need OneBoundedSetup to test the supervision. This is a very low level testing facility, but I don't think this is needed in this case.